### PR TITLE
Update vulnarable SkiaSharp version

### PR DESCRIPTION
**Pull request by Bykiev**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/Fonts/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

Update vulnarable SkiaSharp version to the latest. Also update BenchmarkDotNet to latest version

### CODE REVIEW

File: tests/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks.csproj [hunk 1]
```diff
@@ -30,7 +30,7 @@
   </Choose>
   
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" />
-    <PackageReference Include="SkiaSharp" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.12" />
```
Comment (JimBobSquarePants) [inline tests/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks.csproj:33]: You don't want the version number here. That's handled in the targets file.

Comment (Bykiev) [reply tests/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks.csproj:33]: I believe it was added by Nuget, will fix it

Comment (JimBobSquarePants) [reply tests/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks/SixLabors.Fonts.Benchmarks.csproj:33]: Thanks!

**Review by JimBobSquarePants — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**